tests/ui/print_literal.rs

```diff
@@ -1,5 +1,5 @@
 #![warn(clippy::print_literal)]
-#![allow(clippy::uninlined_format_args)]
+#![allow(clippy::uninlined_format_args, clippy::literal_string_with_formatting_arg)]
```

Comment:
Why does it have to be allowed in these files? Does it also lint strings in `print!` macros? :thinking: